modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -49,6 +49,20 @@ const List = ({items, schema}) => {
 	);
 };
 
+const Title = ({item, title, titleRenderer}) => {
+	const TitleRendererComponent = titleRenderer.component;
+
+	if (TitleRendererComponent) {
+		return <TitleRendererComponent itemData={item} />;
+	}
+
+	if (title) {
+		return <ClayList.ItemTitle>{item[title]}</ClayList.ItemTitle>;
+	}
+
+	return null;
```

Comment:
I'll keep this in mind, but I do like having a fallback so the UI still looks "normal" with the description vertically centered.

Since the `Title` component is only used within this file, I'll keep this as is for now. In the future I'll look into moving conditionals outside of the component to hide the component. (I also don't want to delay this PR further with another ci test run 😂 )